language/German/strings.po

```diff
@@ -456,11 +462,11 @@ msgstr "Geschwindigkeit"
 
 #: src/frontend/mame/ui/submenu.cpp:60
 msgid "Adjust speed to match refresh rate"
-msgstr ""
+msgstr "Geschwindigkeit an die Bildwiederholfrequenz anpassen"
 
 #: src/frontend/mame/ui/submenu.cpp:61
 msgid "Low latency"
-msgstr ""
+msgstr "Geringe Latenzzeit"
```

Comment:
"niedrige Latenz" would probably be better in that context

Reply:
Fixed